aat/src/aat/resources/features/F-079/F-079.feature

```diff
@@ -0,0 +1,43 @@
+@F-079 
+Feature: CCD Definition Store Api :: GET /api/data/case-type/{ctid}
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-508 #RDM-6858
+  Scenario: must return the case type for an appropriate request
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid case type id]
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
+    Then a positive response is received
+    And the response [has the 200 OK code]
+    And the response has all other details as expected
+
+  @S-509  @Ignore # Response code mismatch, expected: 401, actual: 403 RDM-6628
+  Scenario: must return 401 when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains an invalid authentication credentials]
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response [contains 401 unauthorised code]
+    And the response has all the details as expected
+
+  @S-510 @Ignore # Response code mismatch, expected: 403, actual: 200 RDM-7562
+  Scenario: must return 403 when request provides authentic credentials without authorised access to the operation 
+    Given a user with [an active profile in CCD, and insufficient privilege to the case type]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
+    Then a negative response is received
+    And the response [contains 403 forbidden code]
+    And the response has all the details as expected
+
+  @S-507
+  Scenario: must return 404 when request provides an invalid case type id
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [fetch a Case Type Schema] operation of [CCD Definition Store]
```

Comment:
add a line about the request to describe what makes this scenario different to return a 404.